__tests__/app.test.js

```diff
@@ -74,12 +74,22 @@ describe('/api/articles/:article_id', () => {
                     body: 'I find this existence challenging',
                     created_at: '2020-07-09T20:11:00.000Z',
                     votes: 100,
-                    article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+                    article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+                    comment_count: 11
```

Comment:
We shouldn't be amending old test for this challenge. We should be adding a new test and our test suite should be robust enough that our old tests still pass. If you had used toMatchObject here instead of toEqual our test suite would have been more robust and wouldn't have required updating